src/m365/planner/commands/tenant/tenant-settings-list.spec.ts

```diff
@@ -0,0 +1,90 @@
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import appInsights from '../../../../appInsights';
+import auth from '../../../../Auth';
+import { Logger } from '../../../../cli';
+import Command from '../../../../Command';
+import request from '../../../../request';
+import { sinonUtil } from '../../../../utils';
+import commands from '../../commands';
+const command: Command = require('./tenant-settings-list');
+
+describe(commands.TENANT_SETTINGS_LIST, () => {
```

Comment:
I think we are missing test to check the standard
`command correctly handles reject request` and `supports debug mode`

Reply:
Thought `supports debug mode` was not needed since this command doesn't have any options. Anyway I added the test. ☺️